webapp/sources/rudder/rudder-core/src/test/resources/configuration-repository/techniques/ncf_techniques/technique_any/2.0/technique.ps1

```diff
@@ -7,14 +7,24 @@
       [string]$techniqueName,
       [parameter(Mandatory=$true)]
       [string]$Version,
-      [switch]$auditOnly
+      [Rudder.PolicyMode]$policyMode
   )
   $reportIdBase = $reportId.Substring(0,$reportId.Length-1)
-  $local_classes = New-ClassContext
+  $localContext = [Rudder.Context]::new()
+  $localContext.Merge($system_classes)
   $resources_dir = $PSScriptRoot + "\resources"
 
   $reportId=$reportIdBase+"id"
-
-  $local_classes = Merge-ClassContext $local_classes $(Package-Install-Version -PackageName "$($node.properties[apache_package_name])" -PackageVersion "2.2.11" -componentName "Test component$&é)à\'`"" -Report:$true -reportId $reportId -techniqueName $techniqueName -auditOnly:$auditOnly).get_item("classes")
-
+  $reportParams = @{
+    ClassPrefix = "package_install_version_${node_properties_apache_package_name_}"
```

Comment:
i don't know what this syntax ought to mean, but i'm sure it won't expand properly